### ISSUE TITLE
Home Assistant docker

### ISSUE THREAD

**Issue by pvizeli** [mjg59/python-broadlink#202]
https://github.com/mjg59/python-broadlink/blob/master/setup.py#L10-L13
They is a old module and inside docker image it install a newer version they is requested by a other packages.

Is posstible to update this and maybe set the correct requirements without this import check?

**Comment by effelle** [comment]
Please be advice that `broadlink` package needs `gcc` in order to install the latest `pycryptodome` or it will fail:
```
apk update
apk fetch build-base
apk add build-base
pip3 install --upgrade broadlink 
```

**Comment by pvizeli** [comment]
Yes, that will be in... -> https://github.com/home-assistant/hassio-homeassistant/blob/master/generic/Dockerfile
